formlibrary/signals.py

```diff
@@ -0,0 +1,8 @@
+from django.dispatch import receiver
+from django.db.models.signals import pre_save
+from .models import CustomForm
+
+
+@receiver(pre_save, sender=CustomForm)
+def pre_save_handler(sender, instance, *args, **kwargs):
+    instance.full_clean()
```

Comment:
I guess it's a thing about preference, but to follow better the flow, I would have the `full_clean()` call inside the `save()`. That's how we are doing it in our microservices and the recommended way in the Django documentation.

Reply:
It's not like that in the `contacts`
https://github.com/Humanitec/ActivityAPI/blob/master/contact/signals.py#L8

Reply:
Neither `appointments`
https://github.com/Humanitec/ActivityAPI/blob/master/appointment/signals.py#L8

Reply:
Neither in `documents-service`
https://github.com/Humanitec/documents-service/blob/master/documents/signals.py#L9

Reply:
Yeah.. just this guy: https://github.com/Humanitec/time_tracking_service/blob/master/timetracking/models.py#L43